### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Keycloak/Admin UI](https://hosted.weblate.org/projects/keycloak/admin-ui/).

It also includes following components:

* [Keycloak/Theme base/account](https://hosted.weblate.org/projects/keycloak/theme-baseaccount/)

* [Keycloak/Theme base/admin](https://hosted.weblate.org/projects/keycloak/theme-baseadmin/)

* [Keycloak/Theme base/email](https://hosted.weblate.org/projects/keycloak/theme-baseemail/)

* [Keycloak/Account UI](https://hosted.weblate.org/projects/keycloak/account-ui/)

* [Keycloak/Theme base/login](https://hosted.weblate.org/projects/keycloak/theme-baselogin/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties [hunk 1]
```diff
@@ -937,3 +937,152 @@ cibaIntervalHelp=Die Mindestzeit in Sekunden, die das Endgerät (CD) zwischen Ab
 rolesPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
 identityProvidersPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
 groupsPermissionsHint=Legt fest, ob feingranulare Berechtigungen für diese Rolle aktiv sein sollen. Wird diese Option deaktiviert, werden alle aktuell vorhandenen feingranularen Berechtigungen gelöscht.
+clientAuthorization=Autorisierung
+path=Pfad
+pairwiseSubAlgorithmSalt.label=Salz
```
Comment (ahus1) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:942]: IMHO "Salt" is a fixed term in hashing / encryption, and shouldn't be translated.

```suggestion
pairwiseSubAlgorithmSalt.label=Salt
```

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties [hunk 2]
```diff
@@ -206,3 +206,11 @@ samlRoleListScopeConsentText=Meine Rollen
 offlineAccessScopeConsentText=Offline Zugriff
 profileScopeConsentText=Benutzerprofil
 emailScopeConsentText=E-Mail Adresse
+organizations=Organisationen
+organizationDescription=Unternehmen ansehen, denen du beigetreten bist
+emptyUserOrganizations=Keine Unternehmen
+emptyUserOrganizationsInstructions=Du bist bisher noch keinem Unternehmen beigetreten.
+searchOrganization=Nach Unternehmen suchen
+organizationList=Liste an Unternehmen
```
Comment (ahus1) [inline js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:214]: IMHO we should translate it to "Organisation"/"Organisationen" consistently, and not switch to "Unternehmen" in some places.

File: js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties [hunk 3]
```diff
@@ -937,3 +937,152 @@ cibaIntervalHelp=Die Mindestzeit in Sekunden, die das Endgerät (CD) zwischen Ab
...
+importRole=Rolle importieren
+authenticatorAttachment.platform=Plattform
+sslType.external=Externe Anfragen
+showMetaData=Metadaten anzeigen
+deleteNode=Node löschen?
+addAaguids=AAGUID hinzufügen
+createUser=Benutzer erstellen
+refreshTokens=Tokens aktualisieren
+kc.identity.authc.method=Authentifizierungsmethode
+clientProfiles=Kunden-Profile
```
Comment (ahus1) [inline js/apps/admin-ui/maven-resources-community/theme/keycloak.v2/admin/messages/messages_de.properties:1017]: "Client" might again be a untranslated term in the context of Keycloak, as it defines an application that is set up to work with Keycloak.

**Review by ahus1 — CHANGES_REQUESTED** [review]
Thank you @Lxchnxr for these translations! See below for some comments. Please discuss here, and update the translations in Weblate. 

Also pinging @robson90 to provide his feedback on this as well, and possibly configure the untranslatable terms.

File: js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties (hunk 2 above)
Comment (robson90) [reply js/apps/account-ui/maven-resources-community/theme/keycloak.v3/account/messages/messages_de.properties:214]: I would go with "Organisation".

**Comment by robson90** [comment]
@ahus1 what about linking this page: https://hosted.weblate.org/browse/keycloak/glossary/
Inside the Readme: https://github.com/keycloak/keycloak/blob/main/docs/translation.md ?

We should discuss, if we want to add more to the glossary.

**Comment by ahus1** [comment]
@robson90 - I changed "Organisations" auf "Organisationen". Now it should be ready to merge.

@jonkoops - this PR shows nicely how this creates a commit per language. The downside: You don't see any more who change what in the Git history, still that information is available in Weblate.

**Review by ahus1 — APPROVED** [review]

**Comment by ahus1** [comment]
@robson90 - 

> what about linking this page: https://hosted.weblate.org/projects/keycloak/glossary/de/
Inside the Readme: https://github.com/keycloak/keycloak/blob/main/docs/translation.md ?

Sounds like a good idea to me. It might be worth explaining a bit more what a Glossary is and what we use if for in this context.

**Pull request merged**